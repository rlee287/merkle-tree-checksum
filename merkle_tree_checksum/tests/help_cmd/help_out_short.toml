bin.name = "merkle_tree_checksum"
args = "-h"
status.code = 1
fs.sandbox = true