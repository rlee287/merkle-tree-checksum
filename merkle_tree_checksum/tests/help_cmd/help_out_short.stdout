Computes a Merkle Tree hash over the given files

Usage: merkle_tree_checksum [OPTIONS] <COMMAND>

Commands:
  generate-hash  Generates Merkle tree hashes
  verify-hash    Verify Merkle tree hashes
  help           Print this message or the help of the given subcommand(s)

Options:
  -q, --quiet...     Print less text
  -j, --jobs <jobs>  Specify size of thread pool for hashing (set to 0 to disable) [default: 4]
  -h, --help         Print help (see more with '--help')
  -V, --version      Print version

Supported hash functions are the SHA2 family, the SHA3 family, Blake2b/Blake2s, Blake3, and CRC32.
